dlpy/layers.py

```diff
@@ -2200,6 +2200,12 @@ class Segmentation(Layer):
         Specifies the factor used to scale target values for a segmentation layer.
     src_layers : iterable Layer, optional
         Specifies the layers directed to this layer.
+    output_image_type: string, optional
+        Specifies the output image type of this layer.
+        possible values: [ WIDE, PNG, BASE64 ]
```

Comment:
base64 is supported in 3.5 or 4.0.1?

Reply:
Both PNG and BASE64 are supported in 4.0.1.